src/platform_impl/windows/dark_mode.rs

```diff
@@ -141,24 +140,17 @@ fn should_apps_use_dark_mode() -> bool {
     lazy_static! {
         static ref SHOULD_APPS_USE_DARK_MODE: Option<ShouldAppsUseDarkMode> = {
             unsafe {
-                const UXTHEME_SHOULDAPPSUSEDARKMODE_ORDINAL: WORD = 132;
+                const UXTHEME_SHOULDAPPSUSEDARKMODE_ORDINAL: PSTR = PSTR(132 as *mut u8);
 
-                let module = libloaderapi::LoadLibraryA("uxtheme.dll\0".as_ptr() as _);
+                let module = LoadLibraryA(PSTR("uxtheme.dll\0".as_ptr() as *mut u8));
```

Comment:
`PSTR` also has an `IntoParam` implementation, but I agree there should probably be a zero-cost constructor that accepts a byte-literal and ensures it has a trailing null character - I think `widestring` has one too.